### PR TITLE
enable: ingress v1 in test clusters

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -61,6 +61,11 @@ skipper_ingress_max_replicas: "50"
 skipper_ingress_min_replicas: "3"
 skipper_ingress_cpu: "1000m"
 skipper_ingress_memory: "1Gi"
+{{if eq .Cluster.Environment "test"}}
```
Comment (AlexanderYastrebov) [inline cluster/config-defaults.yaml]: I have noticed that we mostly check `production` elsewhere but I guess there is no difference if we revert the condition to disable for `production`?..

Comment (szuecs) [reply cluster/config-defaults.yaml]: we would switch e2e tests to v1 if we change to disable production, maybe it's better to do it as you pointed out

Comment (szuecs) [reply cluster/config-defaults.yaml]: updated the PR

**Comment by mikkeloscar** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
Do we want to bump https://github.com/zalando-incubator/kubernetes-on-aws/blob/4cac0485e475ed02a2ce031cf597301a55a9201d/cluster/manifests/skipper/routesrv-deployment.yaml#L51 or we do it separately?

**Comment by szuecs** [comment]
:+1:

**Pull request merged**